tooling/templatize/main.go

```diff
@@ -126,3 +130,34 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+func (opts *GenerationOptions) ExecuteTemplate(ctx context.Context) error {
+	cfg := config.NewConfigProvider(opts.ConfigFile, opts.Region, opts.User)
+	vars, err := cfg.GetVariables(ctx, opts.Cloud, opts.DeployEnv)
+	if err != nil {
+		return err
+	}
+	// print the vars
+	for k, v := range vars {
+		println(k, v)
```

Comment:
It's encouraged to use fmt.Printline. Please take a look https://go.dev/ref/spec#Bootstrapping